options/locale/locale_en-US.ini

```diff
@@ -594,20 +594,20 @@ Content = Content
 SSPISeparatorReplacement = Separator
 SSPIDefaultLanguage = Default Language
 
-require_error = ` cannot be empty.`
-alpha_dash_error = ` should contain only alphanumeric, dash ('-') and underscore ('_') characters.`
-alpha_dash_dot_error = ` should contain only alphanumeric, dash ('-'), underscore ('_') and dot ('.') characters.`
-git_ref_name_error = ` must be a well-formed Git reference name.`
-size_error = ` must be size %s.`
-min_size_error = ` must contain at least %s characters.`
-max_size_error = ` must contain at most %s characters.`
-email_error = ` is not a valid email address.`
-url_error = `"%s" is not a valid URL.`
-include_error = ` must contain substring "%s".`
-glob_pattern_error = ` glob pattern is invalid: %s.`
-regex_pattern_error = ` regex pattern is invalid: %s.`
-username_error = ` can only contain alphanumeric characters ('0-9','a-z','A-Z'), dash ('-'), underscore ('_') and dot ('.'). It cannot begin or end with non-alphanumeric characters, and consecutive non-alphanumeric characters are also forbidden.`
-invalid_group_team_map_error = ` mapping is invalid: %s`
+require_error = " cannot be empty."
+alpha_dash_error = " should contain only alphanumeric, dash ('-') and underscore ('_') characters."
+alpha_dash_dot_error = " should contain only alphanumeric, dash ('-'), underscore ('_') and dot ('.') characters."
+git_ref_name_error = " must be a well-formed Git reference name."
+size_error = " must be size %s."
+min_size_error = " must contain at least %s characters."
+max_size_error = " must contain at most %s characters."
+email_error = " is not a valid email address."
+url_error = "\"%s\" is not a valid URL."
```

Comment:
Wrong.


Please fully understand the problem first.


<img width="965" height="1144" alt="Image" src="https://github.com/user-attachments/assets/e65fc24e-c762-42e8-b0ce-dcb2a5e77262" />

Reply:
It was converted according to the first comment here.
<img width="674" height="72" alt="image" src="https://github.com/user-attachments/assets/e3771546-fa5a-4c51-8910-c1f674d58c47" />

Reply:
No, it's wrong. INI package can't parse it. See the test cases